### PR TITLE
 🎂 1.4.0

### PR DESCRIPTION
**Pull request by EverlastingBugstopper**
https://github.com/cloudflare/wrangler/milestone/13

https://github.com/issues?utf8=%E2%9C%93&q=repo%3Acloudflare%2Fwrangler+merged%3A%3E%3D2019-09-17

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,125 @@
...
+
+    - [Create a new project from scratch](https://developers.cloudflare.com/workers/sites/start-from-scratch)
+    - [Deploy a pre-existing static site project](https://developers.cloudflare.com/workers/sites/start-from-existing)
+    - [Add static assets to a pre-existing Workers project](https://developers.cloudflare.com/workers/sites/start-from-worker)
+
+    For more details on how Workers Sites works with Wrangler, check out [the documentation](https://developers.cloudflare.com/workers/sites/reference). We also have a brand new [tutorial](https://developers.cloudflare.com/workers/tutorials/deploy-a-react-app) to help you learn the Workers Sites workflow, by deploying a React application!
+
+    Workers Site has been a heroic effort by the entire Workers Developer Experience team, comprising of Wrangler updates, new [project templates](https://github.com/cloudflare/worker-sites-template), and [open-source packages](https://github.com/cloudflare/kv-asset-handler). We're super excited about the future that Workers Sites represents, where static sites and serverless functions can work together to build powerful, cutting-edge applications.
```
Comment (ashleymichal) [inline CHANGELOG.md]:
```suggestion
    Workers Sites has been a heroic effort by the entire Workers Developer Experience team, comprising of Wrangler updates, new [project templates](https://github.com/cloudflare/worker-sites-template), and [open-source packages](https://github.com/cloudflare/kv-asset-handler). We're super excited about the future that Workers Sites represents, where static sites and serverless functions can work together to build powerful, cutting-edge applications. 
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,125 @@
...
+
+    [phayes]: https://github.com/phayes
+    [pull/633]: https://github.com/cloudflare/wrangler/pull/633
+
+- ### Fixes
+
+  **Ensure we install and cache the latest version of cargo-generate and wasm-pack if user has an outdated cargo installed version - [EverlastingBugstopper], [issue/666] [pull/726]**
+
+    Wrangler orchestrates a few other tools under the hood, notably [`wasm-pack`] and [`cargo-generate`]. We use a library called [`binary-install`] to fetch and cache binaries we download. However, to avoid downloading unecessarily, we first check if the user has a copy locally on their machine that they had `cargo install`'d. We had a bug where in this logic branch, we *didn't* check that the local version was the most up-to-date version. This meant that users who had an older installed version may run into errors when wrangler expected to use features of a newer version of that tool. This PR adds the logic to check for the version and will install and cache a newer version for wrangler to use (leaving your local version as is!).
```
Comment (ashleymichal) [inline CHANGELOG.md]: just curious, why square brackets around package names?

File: CHANGELOG.md (hunk 1 above)
Comment (signalnerve) [reply CHANGELOG.md]: oop

File: CHANGELOG.md (hunk 2 above)
Comment (signalnerve) [reply CHANGELOG.md]: ah! i think ashley meant for these to be markdown links - i'll add

**Pull request merged**